### PR TITLE
Replace master reference with main for ironic-image

### PR DESCRIPTION
**Pull request by kashifest**

### CODE REVIEW

File: scripts/init-repo.sh [hunk 1]
```diff
@@ -14,9 +14,9 @@ setup_repo () {
     git remote add upstream git@github.com:"${2}"/"${1}".git
     git remote set-url --push upstream no_push
     git remote -v
-    # Update "master" on Nordix
+    # Update "main" on Nordix
```
Comment (Rozzii) [inline scripts/init-repo.sh:17]: I think at this point in time when not all of the repos are configured to have a main branch  yet this change might cause problems, but I don't see this used in CI so I assume it is okay to change already.

File: wow/github-workflow.md [hunk 2]
```diff
@@ -193,10 +193,10 @@ Instead the code is merged with the internal pull request.
 
 ## How to backport
 
-Sometimes you may need to backport a commit (e.g. bug fix) from a master branch
+Sometimes you may need to backport a commit (e.g. bug fix) from a main/master branch
```
Comment (Rozzii) [inline wow/github-workflow.md:196]: There is a leftover "master".

File: wow/github-workflow.md [hunk 3]
```diff
@@ -193,10 +193,10 @@ Instead the code is merged with the internal pull request.
...
 into a stable release branch. This involves a couple of steps as described below.
 In this example, we will use `release-0.3` as the stable branch in to which we will backport a
-specific commit from the `master` branch.
+specific commit from the `main`/`master` branch.
```
Comment (Rozzii) [inline wow/github-workflow.md:199]: There is a leftover "master".

File: README.md [hunk 4]
```diff
@@ -8,7 +8,7 @@ Set of internal tools for Airship and metal3 development
 make setup-local-repos
 ```
 
-## Update the nordix master branches
+## Update the nordix master/main branches
```
Comment (Rozzii) [inline README.md:11]: As you have modified scripts/init-repo.sh it is only main not master/main

**Review by Rozzii — CHANGES_REQUESTED** [review]

File: scripts/init-repo.sh (hunk 1 above)
Comment (Rozzii) [inline scripts/init-repo.sh:17]: Check the end of this file, this change is only applicable after both metal3-dev-env and metal3-docs are configured to use "main" as primary branch.

**Review by Rozzii — CHANGES_REQUESTED** [review]

File: wow/github-workflow.md (hunk 2 above)
Comment (kashifest) [reply wow/github-workflow.md]: Done

File: wow/github-workflow.md (hunk 3 above)
Comment (kashifest) [reply wow/github-workflow.md]: Done

File: README.md (hunk 4 above)
Comment (kashifest) [reply README.md]: Done

File: scripts/init-repo.sh (hunk 1 above)
Comment (kashifest) [reply scripts/init-repo.sh:17]: Lets keep it since I will be doing the changes soon.

**Review by Rozzii — APPROVED** [review]

**Pull request merged**